### DISCUSSION TITLE
Define __call magic method on trait is accesible from JavaScript

### DISCUSSION THREAD

**Discussion by rafaelindriago** [livewire/livewire#4748] [3. Feature ideas]
I try to define the magic method in a trait, and then i try to call it from JS with:

`Livewire.find(id).__call();`

And the method is called 😮

Even throw an error because the arguments "method" and "params" are not passed 😳

`public function __call($method, $params) { }`

I know maybe this is not useful, but i think this could be a problem for someone who needs to define magic methods, maybe reject calls for method that starts with __ 🤔

**Comment by joshhanley** [comment]
@rafaelindriago is this something you need help with, or are you trying to raise this as a bug or a feature request? If so, please choose one of them (help, bug, feature ideas) for the category, rather than general 🙂

**Reply by rafaelindriago** [reply]
@joshhanley sorry for use the wrong category, i suppose this is a recommendation, i think the magic methods of PHP must not be accessible to call from JavaScript with Livewire

**Reply by joshhanley** [reply]
@rafaelindriago all good, just wasn't clear, thanks for updating it 🙂

Yeah good call, I don't think it should be callable from javascript either.

PR's are welcome! 🙂
